backend/src/libs/packages/token/token.package.ts

```diff
@@ -17,8 +18,17 @@ class Token implements IToken {
       .sign(this.createSecret());
   }
 
-  public verify(): Promise<void> {
-    return Promise.resolve();
+  public async verify(token: string): Promise<JWTPayload> {
+    try {
+      const { payload } = await jwtVerify(token, this.createSecret());
+
+      return payload;
+    } catch {
+      throw new HttpError({
+        message: 'Invalid token',
+        status: HttpCode.UNAUTHORIZED,
+      });
+    }
```

Comment:
```suggestion
```
do not an aditional try catch block

Reply:
Am I supposed to add handler for JWSInvalid exception into common error handler or just handle it inside auth plugin and throw HttpError?

Reply:
do not handle it for it. we will do it in the separate task